ccmlib/cluster.py

```diff
@@ -41,14 +41,19 @@ def __init__(self, path, name, partitioner=None, install_dir=None, create_direct
 
         # This is incredibly important for
         # backwards compatibility.
-        if 'cassandra_version' in kwargs:
-            version = kwargs['cassandra_version']
-        if 'cassandra_dir' in kwargs:
-            install_dir = kwargs['cassandra_dir']
+        version = kwargs.get('cassandra_version', version)
+        install_dir = kwargs.get('cassandra_dir', install_dir)
+        docker_image = kwargs.get('docker_image')
         if create_directory:
             # we create the dir before potentially downloading to throw an error sooner if need be
             os.mkdir(self.get_path())
 
+        if docker_image:
+            self.__install_dir = None
+            self.__version = '3.0'
```

Comment:
meanwhile we'll hardcode this version, until we'll get a reasonable way to pull the version out of the image

Reply:
what is this version about?

Reply:
Across ccm there lots of place logic is based on the version. In other options we have the version available on a file, it's less critical, but would be nice to get the version at an early stage somehow (maybe like I'm getting the scylla.yaml running the image temporary just to run `scylla --version`